javalib-ext-dummies/src/main/scala/java/text/DecimalFormatSymbols.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package java.text
+
+import java.util.Locale
+
+/** Dummy implementation of `DecimalFormatSymbols`.
+ *
+ *  It is even worse than most other dummies, in the sense that it
+ *  special-cases the locales that we use in our tests (`FormatterTestEx`).
+ *  It is incorrect for most locales.
```

Comment:
In the spirit of what we discussed on the last PR: Make it fail (hard!) for locales it doesn't know? So it either is correct or throws an `Error`?